e2e/motoko/Main.mo

```diff
@@ -1,202 +1,214 @@
+            label validate {
+                switch result {
+                    case (#ok(#Object fields)) {
+                        for ((key, val) in fields.vals()) {
+                            switch (key, val) {
+                                case ("result", #String val) {
+                                    assert Text.startsWith(val, #text "0x");
+                                    break validate;
+                                };
+                                case _ {};
                             };
-                            case _ {};
                         };
                     };
+                    case _ {};
                 };
-                case _ {};
+                Debug.trap(debug_show result);
             };
-            Debug.trap(debug_show result);
-        };
 
-        // Candid-RPC methods
-        type RpcResult<T> = { #Ok : T; #Err : EvmRpcCanister.RpcError };
-        type MultiRpcResult<T> = {
-            #Consistent : RpcResult<T>;
-            #Inconsistent : [(EvmRpcCanister.RpcService, RpcResult<T>)];
-        };
+            // Candid-RPC methods
+            type RpcResult<T> = { #Ok : T; #Err : canister.RpcError };
+            type MultiRpcResult<T> = {
+                #Consistent : RpcResult<T>;
+                #Inconsistent : [(canister.RpcService, RpcResult<T>)];
+            };
 
-        func assertOk<T>(method : Text, result : MultiRpcResult<T>) {
-            switch result {
-                case (#Consistent(#Ok _)) {};
-                case (#Consistent(#Err err)) {
-                    Debug.trap("received error for " # method # ": " # debug_show err);
-                };
-                case (#Inconsistent(results)) {
-                    for ((service, result) in results.vals()) {
-                        switch result {
-                            case (#Ok(_)) {};
-                            case (#Err(err)) {
-                                Debug.trap("received error in inconsistent results for " # method # ": " # debug_show err);
+            func assertOk<T>(method : Text, result : MultiRpcResult<T>) {
+                switch result {
+                    case (#Consistent(#Ok _)) {};
+                    case (#Consistent(#Err err)) {
+                        Debug.trap("received error for " # method # ": " # debug_show err);
+                    };
+                    case (#Inconsistent(results)) {
+                        for ((service, result) in results.vals()) {
+                            switch result {
+                                case (#Ok(_)) {};
+                                case (#Err(err)) {
+                                    Debug.trap("received error in inconsistent results for " # method # ": " # debug_show err);
+                                };
                             };
                         };
                     };
                 };
             };
-        };
 
-        let candidRpcCycles = 1_000_000_000_000;
```

Comment:
Why is there so much code in red and identical-looking code in green below..?

Reply:
This is due to a change in indentation (modified the E2E test to cover both the standard and fiduciary canisters).